server.py

```diff
@@ -536,6 +536,53 @@ def bus_hail():
 
   return jsonify(hail)
 
-
+@app.route('/woah', methods=['POST'])
+def woah():
+  '''
+    Basic endpoint to catch/throw "the woah"
+    ---
+    tags:
+      - ASCII
+    parameters:
+    - in: body
+      schema:
+        id: woah
+        type: object
+        properties:
+          value:
+            required: True
+            type: string
+            example: 'catch the whoa!'
+            description: a command to tell whether or not to "catch the woah"
+    responses:
+      200:
+        description: A JSON object of us catching the woah
+        content:
+          text/json:
+            schema:
+              properties:
+                message:
+                  type: string
+                  example: '        😲
+                                  ✊|
+                                    |🤚
+                                   / \\
+                                  /    \\'
+                  description: Caught the woah!
+    
+  '''
+  value = request.form.get('value', '')
+  catch = '''
+        😲
+       ✊|
+         |🤚
+        / \\
+      /    \\
+  '''
+  if "catch" in value:
+    return jsonify(catch)
+  else:
```

Comment:
It would be nice to have a case for when no data was sent with the request and another one for if the "value" key was not used in the request.

Reply:
Since the `GET` request can only detect if the `value` key is present or not, I only have two cases for error handling now.